spec/player_spec.rb

```diff
@@ -0,0 +1,28 @@
+require_relative '../lib/player.rb'
+
+describe Player do
+  let(:player1) { Player.new('Mercedes', 'X') }
+
+  it 'has the symbol and name attributes' do
+    expect(player1).to have_attributes(name: 'Mercedes', symbol: 'X')
+  end
```

Comment:
## Negative scenarios
- Cover positive and negative scenarios ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/commonsense.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/)) Example: there is a spec for correct usage of the method \[e.g. `it "creates object if all required attributes are passed"`\] and also there is spec for the wrong usage of a method \[e.g. `it "does not create the object if required attributes are missing"`\].

  - [ ] Thus, please ads a negative scenario

Reply:
We gave negative scenarios to initialize and array player methods. We expect it to be enough. If it is not, please let us know.